src/solvers/parabolic_solvers.jl

```diff
@@ -0,0 +1,35 @@
+"""
+  struct BR1 end
+
+The Bassi-Rebay flux from "A High-Order Accurate Discontinuous Finite Element Method for
+the Numerical Solution of the Compressible Navier-Stokes Equations" by Bassi, Rebay (1997).
+
+https://doi.org/10.1006/jcph.1996.5572
+"""
+struct BR1 end
```

Comment:
```suggestion
"""
The classical BR1 flux from

- F. Bassi, S. Rebay (1997)
  A High-Order Accurate Discontinuous Finite Element Method for
  the Numerical Solution of the Compressible Navier-Stokes Equations
  [DOI: 10.1006/jcph.1996.5572](https://doi.org/10.1006/jcph.1996.5572)
"""
struct BR1 end
```
This formatting is consistent with the other reference formattings we use.